src/main/ContinuousIntegrationServer.java

```diff
@@ -25,13 +34,24 @@ public void handle(String target,
         baseRequest.setHandled(true);
 
         System.out.println(target);
+        
+
 
         // here you do all the continuous integration tasks
         // for example
         // 1st clone your repository
+        File localDirectory = new File("TestGitPull\\");
+    	Git git = GitConnector.cloneRepo("https://github.com/DD2480-Group26/DD2480-CI.git",localDirectory);
+    	GitConnector.gitPull(localDirectory, "test");
```

Comment:
should it be called 'test'?

Reply:
should probably be called the same as the branch name, imo. this information should be in one of the argument strings, probably 'baseRequest' or 'request'.